.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: objective-c
+
+# macOS and Xcode Version
+# Xcode 9.2 running on macOS 10.12
+osx_image: xcode9.2
+
+# Build dependencies
+install:
+  - brew install python3
+  - sudo gem install asciidoctor
+  - ( cd External && ./makeAll )
+
+# Cache built deps
+cache:
+  directories:
+    - External
+
+script:
+  - xcodebuild -scheme "MoltenVK (Debug)"
```

Comment:
Probably better to build the Release scheme?

Reply:
I wasn't sure, while release is what most people build, there could be mistakes in code that is only enabled in Debug mode, which would then slip through the CI tests.

Reply:
That's a fair point. I guess Debug mode does give us the best coverage. I'm happy to leave it as Debug mode then.